sycl/test/extensions/properties/properties_kernel_negative_device.cpp

```diff
@@ -16,6 +16,20 @@ template <size_t... Is> struct KernelFunctorWithWGSizeWithAttr {
   }
 };
 
+struct KernelFunctorWithOnlyWGSizeHintAttr {
+  // expected-warning@+1 {{kernel has both attribute 'work_group_size_hint' and kernel properties; conflicting properties are ignored}}
+  void operator() [[sycl::work_group_size_hint(32)]] () const {}
+};
```

Comment:
Is that based on the place where the kernel is submitted?

Reply:
I agree, technically it would be better to do it at submission, but to my knowledge there's no good way to get back to the submission from the level where we can detect this mismatch.